gcov/gcovclient.c

```diff
@@ -0,0 +1,222 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include <errno.h>
+#include <fcntl.h>
+#include <limits.h>
+#include <stdarg.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <unistd.h>
+
+#define MYST_GCOV_STDERR ((FILE*)0x67001b41aafb4224)
```

Comment:
I'd remove this define and include ``<myst/gcov.h>``

Reply:
Added!